frontend/src/components/Displays/RobotDisplays/BatteryStatusDisplay.tsx

```diff
@@ -14,55 +13,36 @@ const StyledTypography = styled(Typography)<{ $fontSize?: 24 | 16 | 18 | 32 | 40
     font-size: ${(props) => props.$fontSize};
 `
 export interface BatteryStatusDisplayProps {
-    battery?: number
-    batteryStatus?: BatteryStatus
+    batteryLevel?: number
     itemSize?: 24 | 16 | 18 | 32 | 40 | 48 | undefined
-    robotStatus: RobotStatus
 }
 
-const BatteryStatusDisplay = ({
-    robotStatus,
-    battery,
-    batteryStatus,
-    itemSize = 24,
-}: BatteryStatusDisplayProps): JSX.Element => {
-    let battery_icon
-    let icon_color: string = tokens.colors.interactive.primary__resting.hex
-    let battery_value: string
+export const BatteryStatusDisplay = ({ batteryLevel, itemSize }: BatteryStatusDisplayProps): JSX.Element => {
+    let iconColor: string = tokens.colors.interactive.primary__resting.hex
 
-    if (!battery) {
-        battery_value = '---%'
-        battery_icon = Icons.BatteryUnknown
-    } else if (robotStatus === RobotStatus.Offline) {
-        battery_value = ''
-        icon_color = tokens.colors.interactive.disabled__text.hex
-        battery_icon = Icons.BatteryUnknown
-    } else {
-        battery_value = `${battery}%`
-        switch (batteryStatus) {
-            case BatteryStatus.Normal:
-                battery_icon = Icons.Battery
-                break
-            case BatteryStatus.Charging:
-                battery_icon = Icons.BatteryCharging
-                break
-            case BatteryStatus.Critical:
-                battery_icon = Icons.BatteryAlert
-                icon_color = tokens.colors.interactive.danger__resting.hex
-                break
+    const getBatteryIcon = (batteryLevel: number) => {
+        switch (true) {
+            case !batteryLevel:
+                return Icons.BatteryUnknown
+            case batteryLevel > 10:
+                return Icons.Battery
+            case batteryLevel <= 10:
+                return Icons.BatteryAlert
```

Comment:
Did we get rid of the BatteryCharging case because we have no way of telling that the battery is charging or an other reason?

Reply:
It's wasn't implemented, it was always being used BatteryStatus.Normal